contracts/vaults/TreasuryVault/RibbonTreasuryVault.sol

```diff
@@ -75,8 +75,8 @@ contract RibbonTreasuryVault is
     // The minimum duration for an option auction.
     uint256 private constant MIN_AUCTION_DURATION = 5 minutes;
 
-    // Maximum number of whitelisted user address.
-    uint256 private constant WHITELIST_LIMIT = 5;
+    // Maximum number of depositor user address.
+    uint256 private constant DEPOSITORS_LIMIT = 30;
```

Comment:
Let's make this configurable by the owner, and public

Reply:
Moved this variable to TreasuryStorage.